main.py

```diff
@@ -8,34 +8,99 @@ def __init__(first_name, last_name, username, password):
 
 # Create a dictionary
 existing_costumers = [
-    Customer ( "first_name" = "Jane", "last_name"= "Cooper", "username"= "Jane_Cooper", "password"= "password000" ),
-    Customer ("first_name" = "Mike", "last_name" = "Frank", "username" = "Mike_Frank", "password" = "password111" )
+    Customer ( "first_name" == "Jane", "last_name"== "Cooper", "username"== "Jane_Cooper", "password"== "password000" ),
+    Customer ("first_name" == "Mike", "last_name" == "Frank", "username" == "Mike_Frank", "password" == "password111" )
 ]
 # Adding two new customers
-existing_customers.append = ("first_name" = "David", "last_name" = "Sharon", "username" = " David_Sharon", "password" = "password222" ) 
-existing_customers.append = ("first_name" = "Eve", "last_name" = "Bron", "username" = "Eve_Baron", "password" = "password333" )
+existing_customers.append = ("first_name" == "David", "last_name" == "Sharon", "username" == " David_Sharon", "password" == "password222" ) 
+existing_customers.append = ("first_name" == "Eve", "last_name" == "Bron", "username" == "Eve_Baron", "password" == "password333" )
```

Comment:
See your earlier lines 11 and 12 where you initialize a `Customer`.  You get the "takes only one argument" error here because you aren't initializing a `Customer` here.